release-team/role-handbooks/enhancements/README.md

```diff
@@ -75,43 +78,123 @@ It is important that this process be followed and documentation remain up-to-dat
+| 1 | Collection | New issues will be created. Take the opportunity to apply correct labels, milestone information, and add it to the `data` tab within the sheet. If it's not being tracked for the current release, adding the information will help a later release team. |
 | 1 | Collection | Weekly Release meetings require updates of current status. Use the `Feature stats` tab to update the release team on counts of enhancements in good and bad progress.|
 | 1 | Collection | Start reminding Issue owners that KEPs are required for each enhancement and that KEPs must be in an implementable state by Enhancement Freeze. |
-| 2-3 | Collection Monitoring and Triage | Stay on top of comments in issues when owners respond. Apply correct labels, milestone information, and add to the sheet if necessary. |
-| 2-3 | Collection Monitoring and Triage | Mark features as "At Risk" if there is no communication or active PRs on the issues coming into Enhancement Freeze. |
+| 2-3 | Collection Monitoring and Triage | Stay on top of comments in issues when owners respond. Apply correct labels, milestone information, update it's status in the sheet if necessary. |
+| 2-3 | Collection Monitoring and Triage | Mark features as `At Risk` if there is no communication, active PRs on the issues, or it is missing other requirements coming into Enhancement Freeze. |
 | 4 | Enhancement Freeze | On Freeze day, send an email to [Kubernetes-Dev](https://groups.google.com/forum/#!forum/kubernetes-dev) that freeze has happened. "We are at X enhancements, and any new ones will require an exception." Examples [1](https://groups.google.com/forum/#!msg/kubernetes-dev/4kefhF6nOFQ/rCOme6vjCAAJ) [2](https://groups.google.com/forum/#!msg/kubernetes-dev/T-kIHtgS5J4/wW73F2HmBAAJ;context-place=msg/kubernetes-dev/T-kIHtgS5J4/ZjKTsCf_AwAJ). |
-| 4 | Enhancement Freeze | Remove any enhancements from the sheet that do not have a KEP in an implementable state. |
-| 4 | Enhancement Freeze | Any enhancements removed from the milestone will now require an exception.  As exception requests come in approve/reject at your descretion and create the exception file in the Release for exceptions Example [1](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.14/exception.yaml). |
+| 4 | Enhancement Freeze | Remove any enhancements that failed to meet the criteria by the Enhancement freeze deadline. Set their status in the sheet to `Removed from Milestone` and use the `Enhancements` -> `Remove Enhancements from Milestone` menu option to move them over to the `Removed from milestone` tab. |
+| 4 | Enhancement Freeze | Any enhancements removed from the milestone will now require an exception.  As exception requests come in approve/reject at your discretion and create the exception file in the Release for exceptions Example [1](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.14/exception.yaml). If a previously removed Enhancement had an acc|
```

Comment:
"If a previously removed Enhancement had an acc|"
Not sure if this is a complete sentence?

Reply:
I think this was a mis copy-paste >_< woops

Reply:
updated it 👍